### ISSUE TITLE
[3.7] Core components loose menu links in Control Panel after manual update

### ISSUE THREAD

**Issue by anabarcellos** [joomla/joomla-cms#16454]
### Steps to reproduce the issue

1. Update from 3.x to 3.7.x manually.  
2. Apply @mbabker's script: https://gist.github.com/mbabker/d7bfb4e1e2fbc6b7815a733607f89281 
==>> Message: Update to 3.7.0 completed successfully. 
OBS: The same result below happens just following the steps from Joomla Docs, without using the mentioned script. 

### Expected result

Go to Control Panel > Components and find all the links you had before version update. 

### Actual result

The links for the core components are not displayed:

_com_banners
com_contact
com_finder
com_joomlaupdate
com_messages
com_newsfeeds
com_redirect
com_search
com_weblinks
_

Information: 
- the components are installed and enabled. 
- data are ok in the #_menu table.
- third part extensions are not affected.
- if I downgrade to 3.6.5, the links appear again.
- the same happens in different admin templates.
- if I install Weblinks from extensions.joomla.org after updating to 3.7.x, it works again, but adding new registers in #_menu.  
**- solution:** UPDATE `#_menu` SET `menutype`='main' WHERE `menutype`='menu'; 

### System information (as much as possible)

Tested updating from different sites:
      - 3.4.5 to 3.7
      - 3.6.5 to 3.7
      - 3.6.5 to 3.7.1
      - 3.6.5 to 3.7.2

SLES 12
PHP Version 5.5.14
Apache/2.4.16 (Linux/SUSE)
MySQL 5.6.31

### Additional comments

**Comment by wojsmol** [comment]
@anabarcellos For same odd reason this file https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_admin/sql/updates/mysql/3.7.0-2017-01-17.sql was not run during update.

**Comment by zero-24** [comment]
> Update from 3.x to 3.7.x manually.

Why do you want to do this?

**Comment by AlexRed** [comment]
@zero-24 in some bad server you can't do the normal update procedure from the "Joomla Update" component. Also in the italian forum we have many user with this problem and now without component menu items.

**Comment by Bakual** [comment]
Manual updating isn't supported. You need to use the Joomla Update component to do it.

**Comment by anabarcellos** [comment]
Really? Since when manual update is not supported? I run a INTRANET project in the biggest bank in Latin America. I can't use Joomla Update. What should I do? Change the CMS? 

@mbabker @brianteeman really? 😥

**Comment by andrepereiradasilva** [comment]
you can use "Upload & Update" feature in Joomla Update component

**Comment by Bakual** [comment]
> Really? Since when manual update is not supported? I run a INTRANET project in the biggest bank in Latin America. I can't use Joomla Update. What should I do? Change the CMS?

Download the package (if the proxy setup doesn't work) and upload it in the Joomla Update component.

The point is that the manual update never worked reliable since not all SQL commands were properly applied. It worked for schema changes, but not for INSERT and UPDATE type of (data) changes.

**Comment by brianteeman** [comment]
Were you not one of the testers of this feature - I was sure you were as it was specifically written for your use case.

**Comment by anabarcellos** [comment]
Perfect! Thank you guys for the information. Sorry for being so dramatic above. I read my post again and it sounds so rude and arrogant. It was not my intention, ok? I was just scared since we have another free software project which can't be updated since 2012 and it it is a big problem for us now.

I was running manual updates since 3.5 and probably something got lost on the way because my 3.7.2 installation has no Joomla Update Component anywhere - not even disabled or in discover. 

@brianteeman after 3.5 I understood that I could not update by file upload and I never heard about this subject anymore. So I didn't test it, sorry. I don't know how I lost this information about 3.6 release. 
I have installed a new site from zero now and it is perfect. Phew! Thank you, guys!!

Thinking that I should configurate a new installation from the beggining again. Even if I solved this component missing issue, how can I know how many other things are not in the right place?  All my new installations are deploys from that default site.  

Thank you again! Not just for the helpful words but for making Joomla each time better for all the users.
